packages/web-components/src/slider/slider.ts

```diff
@@ -0,0 +1,26 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTSlider } from '@microsoft/fast-foundation';
+import type { SliderSize } from './slider.options.js';
+
+/**
+ * The base class used for constructing a fluent-slider custom element
+ * @public
+ */
+export class Slider extends FASTSlider {
+  /**
+   * The size of the slider
+   * @public
+   * @remarks
+   * HTML Attribute: size
+   */
+  @attr
+  public size: SliderSize = 'medium';
```

Comment:
Per @miroslavstastny and prior comments, let's keep the defaults set in CSS only, no need to set the default value here.